test/python/ibmq/test_ibmq_qobj.py

```diff
@@ -13,88 +13,87 @@
 
 import os
 import unittest
+import functools
 from qiskit import (ClassicalRegister, QuantumCircuit, QuantumRegister, compile)
-
 from qiskit import IBMQ, Aer
 from qiskit.qasm import pi
-from ..common import requires_qe_access, JobTestCase, slow_test
+
+from ..common import require_multiple_credentials, JobTestCase, slow_test
+
+# Timeout duration
+TIMEOUT = os.getenv("IBMQ_TIMEOUT", 10)
+
 
 
-class TestIBMQQobj(JobTestCase):
-    """Qiskit backend qobj test. Compares remote simulator as
-       configured in environment variables 'IBMQ_QOBJ_DEVICE',
-       'IBMQ_TOKEN' and 'IBMQ_QOBJ_URL' against local simulator
-       'local_qasm_simulator' as ground truth.
+def once_per_qobj_backend(test):
     """
+    Test Qobj support on all backends claiming to support Qobj.
+    This way VCR creates a single cassette for each test.
+    """
+
+    @require_multiple_credentials
+    @functools.wraps(test)
+    def _wrapper(self, *args, credentials=[], **kwargs):
+        for qe_token, qe_url in credentials:
+            IBMQ.enable_account(qe_token, qe_url)
+        for backend in IBMQ.backends():
+            config = backend.configuration()
+            if config['allow_q_object']:
+                with self.subTest(backend=backend):
+                    backend_test = test if config['simulator'] else slow_test(test)
+                    backend_test(self, backend, *args, **kwargs)
```

Comment:
If possible, I think it's better if we disable the accounts after the test?

Reply:
Agreed, won't hurt.